cmd/node/config/gasSchedules/gasScheduleV2.toml

```diff
@@ -32,8 +32,8 @@
     DataCopyPerByte   = 1000
     PersistPerByte    = 10000
     CompilePerByte    = 300
-    AoTPreparePerByte = 200
-    GetCode           = 100000
+    AoTPreparePerByte = 300
+    GetCode           = 1000000
```

Comment:
10x expensive?

Reply:
yup, according to our tests, those are the correct new values